dev-cpp/antlr-cpp/antlr-cpp-4.13.2.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="The ANTLR 4 C++ Runtime"
+HOMEPAGE="https://www.antlr.org/"
+SRC_URI="https://www.antlr.org/download/antlr4-cpp-runtime-${PV}-source.zip -> ${P}.zip"
```

Comment:
I think the renaming here is unhelpful. It harms provenance to deviate from upstream naming unnecessarily.